### PR TITLE
add setup tools specific to python versions

### PR DESCRIPTION
**Pull request by strongjz**

### CODE REVIEW

**Comment by kaniini** [comment]
Looks okay for a first pass, except that it fails CI.

File: Makefile [hunk 1]
```diff
@@ -127,6 +127,9 @@ $(eval $(call build-package,libarchive,3.6.2-r2))
 $(eval $(call build-package,libuv,1.44.2-r3))
 $(eval $(call build-package,rhash,1.4.3-r3))
 $(eval $(call build-package,cmake,3.24.2-r2))
+$(eval $(call build-package,py3.10-setuptools,67.4.0))
```
Comment (dlorenc) [inline Makefile:130]: I think you're missing the -r0 here

Comment (imjasonh) [inline Makefile:130]:
```suggestion
$(eval $(call build-package,py3.10-setuptools,67.4.0-r0))
```

File: py3.10-setuptools.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    version: 67.4.0
+    epoch: 0
+    description: Easily download, build, install, upgrade, and uninstall Python packages
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: TODO
+          license: ""
```
Comment (imjasonh) [inline py3.10-setuptools.yaml:13]:
```suggestion
          license: MIT
```

File: py3.11-setuptools.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    version: 67.4.0
+    epoch: 0
+    description: Easily download, build, install, upgrade, and uninstall Python packages
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: TODO
+          license: ""
```
Comment (imjasonh) [inline py3.11-setuptools.yaml:13]:
```suggestion
          license: MIT
```

File: Makefile (hunk 1 above)
Comment (strongjz) [reply Makefile]: fixed

File: Makefile [hunk 4]
```diff
@@ -127,6 +127,9 @@ $(eval $(call build-package,libarchive,3.6.2-r2))
 $(eval $(call build-package,libuv,1.44.2-r3))
 $(eval $(call build-package,rhash,1.4.3-r3))
 $(eval $(call build-package,cmake,3.24.2-r2))
+$(eval $(call build-package,py3.10-setuptools,67.4.0-r0))
+$(eval $(call build-package,py3.11-setuptools,67.4.0-r0))
+$(eval $(call build-package,py3-appdirs,1.4.4-r3))
```
Comment (imjasonh) [inline Makefile:132]:
```suggestion
```

my bad

**Review by imjasonh — DISMISSED** [review]

**Pull request closed without merging**